packages/prerender/src/runPrerender.tsx

```diff
@@ -0,0 +1,96 @@
+import fs from 'fs'
+import path from 'path'
+
+import React from 'react'
+
+import babelRequireHook from '@babel/register'
+import prettier from 'prettier'
+import ReactDOMServer from 'react-dom/server'
+
+import { getPaths } from '@redwoodjs/internal'
+import { RedwoodApolloProvider } from '@redwoodjs/web/dist/components/RedwoodApolloProvider'
+
+import { getRootHtmlPath, registerShims, writeToDist } from './internal'
+
+interface PrerenderParams {
+  inputComponentPath: string // usually web/src/{components/pages}/*
+  outputHtmlPath: string // web/dist/{path}.html
+  dryRun: boolean
+}
+
+const rwWebPaths = getPaths().web
+
+// Prerender specific configuration
+// extends projects web/babelConfig
+babelRequireHook({
+  extends: path.join(rwWebPaths.base, '.babelrc.js'),
+  extensions: ['.js', '.ts', '.tsx', '.jsx'],
+  plugins: [
+    ['inline-react-svg'],
+    ['ignore-html-and-css-imports'], // webpack/postcss handles CSS imports
+    [
+      'babel-plugin-module-resolver',
+      {
+        alias: {
+          src: rwWebPaths.src,
+        },
+      },
+    ],
+  ],
+  only: [rwWebPaths.base],
+  ignore: ['node_modules'],
+  cache: false,
```

Comment:
```suggestion
  cache: false,
  overrides: [
    {
       test: ['./web/src/Routes.js', './web/src/Routes.tsx'],
       plugins: undefined
     },
  ]
```

I wonder if something like this could work?

otherwise we can set `process.env.__REDWOOD_PRERENDER = true` before this babel register thing, and then do this in the `babel-preset.js`:

```js
      // Automatically import files in `./web/src/pages/*` in to
      // the `./web/src/Routes.[ts|jsx]` file.
      process.env.__REDWOOD__PRERENDER !== true && {
        test: ['./web/src/Routes.js', './web/src/Routes.tsx'],
        plugins: [
          [
            require('../dist/babelPlugins/babel-plugin-redwood-routes-auto-loader'),
            {
              project,
            },
          ],
        ],
      },

```

Reply:
Danny confirmed that overrides merges. Lame.